python/paddle/jit/sot/opcode_translator/executor/function_graph.py

```diff
@@ -103,14 +106,19 @@
     ],
 ]
 
+InputVariableType = Union[TensorVariable, SymbolicVariable, NumpyArrayVariable]
```

Comment:
```suggestion
GraphNodeVariableType: TypeAlias = Union[TensorVariable, SymbolicVariable, NumpyArrayVariable]
```

这个可以写在 `if TYPE_CHECKING` 下